Source/WebCore/css/CSSProperties.json

```diff
@@ -1489,15 +1489,15 @@
         "background": {
             "codegen-properties": {
                 "longhands": [
+                    "background-color",
```

Comment:
Why move this? The [spec](https://drafts.csswg.org/css-backgrounds-3/#background) says "canonical order: per grammar", and `backgrfound-color` only appears at the end in the `<final-bg-layer>`.

Reply:
If there's only one background-image, then background-color does appear first. The spec is just saying you can't mention multiple background-colors in the shorthand (which makes sense). 

I think this deserves some clarification?

Reply:
Well, even if there is only a single layer, `<bg-layer>` will be omitted but it's still in the grammar, so `background-color` appears last. I agree that "canonical order: per grammar" may not be much clear in these complex cases, but I don't see what backs this:

> Moved background-color to the start of the background shorthand so serialization matches the specification.

Reply:
https://www.w3.org/TR/css-backgrounds-3/#typedef-final-bg-layer lists the color first. That is where I got the idea from. Maybe I am looking in the wrong place.

Reply:
I see your point now.

I’d be happy to move `background-color` back to the end. I suggest we try to resolve this in https://github.com/w3c/csswg-drafts/issues/418 and then change to match at that time. Whoever wrote the WPT test apparently read the specification the same way I did.

Reply:
Note that the WebKit parser does **not** parse a trailing color as a separate final background layer, nor should it. So while the grammar states something like that, it’s not at all consistent with how the values are interpreted. Anyway, I’m willing to change back.